pyvista/_vtk.py

```diff
@@ -297,6 +297,7 @@
         vtkPTSReader,
         vtkSTLReader,
         vtkSTLWriter,
+        vtkTecplotReader,
```

Comment:
How do these changes pass the vtk==8.2 test if this isn't imported for vtk<9?

Reply:
With an install of VTK 8.1.2
```
$ pip list | grep vtk
vtk                           8.1.2
```

I can still import this.
```
>>> from pyvista import _vtk
>>> _vtk.vtkTecplotReader()
(vtkIOGeometryPython.vtkTecplotReader)0x7fbe07fb2360
```

This class appears to predate v9.1 and is imported with `from vtk import *`.